setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
Impressive, that's expected outcome. I'm looking forward having `remote` as separate plugin (the same way we have the `html` plugin.